java/vertx/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <properties>
         <java.version>11</java.version>
-        <vertx.version>4.0.2</vertx.version>
+        <vertx.version>4.0.3</vertx.version>
```

Comment:
Perhaps, I've already asked, but using a range is feasible / recommended ?
`[4.0,4.1)`

/cc @whiplash

Reply:
No, because of `dependencyManagement` section. Using a BOM requires a fixed version if I'm not mistaken. We need it because the version of vertx and netty should be aligned and the BOM does that for us.